lib/over_react.dart

```diff
@@ -27,7 +27,7 @@ export 'package:react/react.dart' show
     SyntheticUIEvent,
     SyntheticWheelEvent;
 
-export 'package:react/react_client.dart' show setClientConfiguration, ReactElement;
+export 'package:react/react_client.dart' show setClientConfiguration, ReactElement, inReactDevMode;
```

Comment:
Why are we exporting this? 

I can't really think of a use-case other consumers would have for this that isn't related to `validateProps.`

I see this flag being a means to have behavioral parity with the prop validation that exists in React JS, and I would be concerned about exposing it for use other use-cases that shouldn't be tied to which React build is being used (and should instead be tied to some other flag that's more appropriate).